lib/PuppeteerSharp.Tests/BrowserContextTests/DefaultBrowserContextTests.cs

```diff
@@ -88,7 +88,7 @@ public async Task PageDeleteCookieAsyncShouldWork()
             });
             Assert.Equal("cookie1=1", await Page.EvaluateExpressionAsync<string>("document.cookie"));
 
-            var cookie = (await Page.GetCookiesAsync()).FirstOrDefault();
+            var cookie = (await Page.GetCookiesAsync()).First();
```

Comment:
no null check after a `FirstOrDefault` implies it can never be empty, and hence should be a `First`